scalafix-cli/src/main/scala/scalafix/internal/cli/ScalafixOptions.scala

```diff
@@ -55,12 +55,28 @@ case class ScalafixOptions(
     sourceroot: Option[String] = None,
     @HelpMessage(
       "java.io.File.pathSeparator separated list of directories or jars containing " +
-        "'.semanticdb' files. The 'semanticdb' files are emitted by the " +
+        "META-INF/semanticdb/**/*.semanticdb files. SemanticDB files are emitted by the " +
         "semanticdb-scalac compiler plugin and are necessary for semantic rules " +
         "like ExplicitResultTypes to function."
     )
     @ValueDescription("entry1.jar:entry2.jar:target/scala-2.12/classes/")
     classpath: Option[String] = None,
+    @HelpMessage(
+      "Same as --classpath except for dependencies that do not need to be compiled with SemanticDB. " +
+        "Must be accompanied with --classpath. " +
+        "This option is required for rules like ExplicitResultTypes to lookup symbol information from the classpath." +
+        "This option is processed with metacp and produced artifacts are globally cached using --metacp-cache-dir"
+    )
+    @ValueDescription("entry1.jar:entry2.jar:target/scala-2.12/classes/")
+    dependencyClasspath: Option[String] = None,
+    @HelpMessage(
+      "Global cache location to persist metacp artifacts produced by analyzing --dependency-classpath. " +
+        "The default location depends on the OS and is computed with https://github.com/soc/directories-jvm " +
+        "using the project name 'semanticdb'. " +
+        "On macOS the default cache directory is ~/Library/Caches/semanticdb. ")
+    metacpCacheDir: Option[String] = None,
+    @HelpMessage("Set this flag to disable parallel processing with metacp.")
+    metacpNoPar: Boolean = false,
```

Comment:
@ShaneDelmore We'll need to set this flag when we'll be upgrading Scalafix internally.